pkg/util/pki/pki.go

```diff
@@ -0,0 +1,94 @@
+package pki
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"crypto/x509"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+var caMap map[string]x509.CertPool
+
+type Pki interface {
+	GetTlsCertPool(caName string) (*x509.CertPool, error)
```

Comment:
potentially we may need a `CertPool` that has all the valid CAs including CAs from system